docs/content/guides/migrations/migrating-airflow-to-dagster.mdx

```diff
@@ -5,223 +5,242 @@ description: "Learn how do a lift-and-shift migration of airflow to Dagster."
 
-```python file=/integrations/airflow/migrate_repo.py
-import os
+@job
+def airflow_taskflow_example():
+    op1 = extract()
+    op2 = transform(op1)
+    load(op2)
 
-from dagster_airflow import make_dagster_definitions_from_airflow_dags_path
 
-migrated_airflow_definitions = make_dagster_definitions_from_airflow_dags_path(
-    os.path.abspath("./dags/"),
+defs = Definitions(
+    jobs=[airflow_taskflow_example]
 )
 ```
 
----
-
-## Step 4: Verify the DAGs are loading
-
-In this step, you'll spin up Dagster's web-based UI, and verify that your migrated DAGs are loading. **Note**: Unless the migrated DAGs depend on no Airflow configuration state or permissions, it's unlikely they'll execute correctly at this point. That's okay - we'll fix it in a bit. Starting the Dagster UI is the first step in our development loop, allowing you to make a local change, view it in the UI, and debug any errors.
-
-1. Run the following to start the UI:
-
-   ```bash
-   dagster dev -f ./migrate_repo.py
-   ```
+Alternatively, to extract the full power of Dagster, you may want to consider leveraging _asset_ paradigm. In this example, we will consolidate the individual operations into a single asset that represents the resulting data of our pipeline.
 
-2. In your browser, navigate to <http://localhost:3001>. You should see a list of Dagster jobs that correspond to the DAGs in your Airflow DagBag.
-
-3. Run one of the simpler jobs, ideally one where you're familiar with the business logic. Note that it's likely to fail due to a configuration or permissions issue.
-
-4. Using logs to identify and making configuration changes to fix the cause of the failure.
-
-Repeat these steps as needed until the jobs run successfully.
-
-### Containerized operator considerations
+```python
+import json
 
-There are a variety of Airflow Operator types that are used to launch compute in various external execution environments, for example Kubernetes or Amazon ECS. When getting things working locally we'd recommend trying to execute those containers locally unless it's either unrealistic or impossible to emulate the cloud environment. For example if you use the K8sPodOperator, it likely means that you will need to have a local Kubernetes cluster running, and in that case we recommend docker's built-in Kubernetes environment. You also need to be able to pull down the container images that will be needed for execution to your local machine.
+from dagster import Definitions, MaterializeResult, define_asset_job, asset
 
-If local execution is impossible, we recommend using Branch Deployments in Dagster+, which is a well-supported workflow for cloud-native development.
+@asset
+def extract_transform_load_asset():
```

Comment:
Would like feedback as to whether or not we want to show both the `@op` approach and the consolidated asset.

Originally I had done three assets: extract, transform, and load -- would that be preferred here?

Reply:
I would choose any asset variation over the standalone `@op` approach. Using `@op` in a graph-backed asset could be a nice middle ground. 

Having three assets also makes sense, as users may want a MaterializeResult for each of them.

Reply:
For now I've removed the `@op` example and replaced it with a 1-to-1 mapping of `@asset`. This is using the implicit asset outs to keep it simple, as I agree that introducing graph-backed right away may be overwhelming.

Reply:
Let's focus on asset!